src/pages/resources/migration/ios/migrate-to-5x.md

```diff
@@ -71,3 +71,11 @@ Once the previous command is complete, run `pod install` or `pod update` to upda
 If you are using Swift Package Manger (SPM) for managing your app dependencies, you can now include the Experience Platform 5.x SDKs either through Xcode UI, or by declaring them as dependencies in the Package.swift project file. For more details, follow the guide for [managing dependencies using Swift Package Manager](../../manage-spm-dependencies.md).
 
 ## Update outdated API references
+
+#### Messaging
+
+Change usages of `Messaging.handleNotificationResponse(_:applicationOpened:withCustomActionId:)` and `Messaging.handleNotificationResponse(_:closure:)` to `Messaging.handleNotificationResponse(_:urlHandler:closure:)`
```

Comment:
Is the API reference in Messaging extension updated with this new API? If so, can you link to that API?

Reply:
@sbenedicadb can we have some comparison and/or examples on how customers should implement these APIs for migration?